release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -3713,3 +3713,27 @@ $ oc adm release info 4.12.19 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-12-20"]
+=== RHSA-2023:3410 - {product-title} 4.12.20 bug fix update
+
+Issued: 2023-06-07
+
+{product-title} release 4.12.20 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:3410[RHSA-2023:3410] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:3409[RHSA-2023:3409] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.12.20 --pullspecs
+----
+
+[id="ocp-4-12-20-bug-fixes"]
+==== Bug fixes
+
+*  Previously, using the mirroring from a registry to disk and specifying in the `imageSetConfig` parameter several digests of the same image, without tags, caused an error because the oc-mirror plugin added a default tag `latest` to all the images (digests). With this update, the oc-mirror plugin now uses a truncated digest of the image eliminanting the error. (link:https://issues.redhat.com/browse/OCPBUGS-13432[*OCPBUGS-13432*])
```

Comment:
>sing the mirroring from a registry to disk and specifying...

I have no idea what this means. Maybe:

> Previously, mirroring from a registry to a disk by using an image set configuration file that specifies several digests of the same image, without tags, caused an error...

Reply:
s/image eliminanting/image, eliminanting (not sure if it is grammatically needed, but it seems clearer to me. Currently, it could sound like the digests is of an image that eliminates the error)